Source/Scene/ShadowMapShader.js

```diff
@@ -143,39 +151,81 @@ define([
             '    return visibility; \n' +
             '} \n' +
             ' \n' +
-            'void main() \n' +
-            '{ \n' +
-            '    czm_shadow_main(); \n' +
-            '    vec3 shadowPosition = v_shadowPosition; \n' +
-            '    // Do not apply shadowing if outside of the shadow map bounds \n' +
-            '    if (any(lessThan(shadowPosition, vec3(0.0))) || any(greaterThan(shadowPosition, vec3(1.0)))) { \n' +
-            '        return; \n' +
-            '    } \n' +
+            'vec2 directionToUV(vec3 d, out vec2 faceUV) { \n' +
             ' \n' +
+            '    vec3 abs = abs(d); \n' +
```

Comment:
Probably not yet. The other types of cube map rendering we'll support will most likely use actual cube maps.